gulp/tasks/build.js

```diff
@@ -5,6 +5,7 @@ var gulp = require( 'gulp' );
 gulp.task( 'build',
   [
     'styles',
+    'ieStyles',
```

Comment:
Perhaps `gulp styles:modern`, `gulp styles:ie`, and `gulp styles` that runs both? Then just run `gulp styles:modern` on watch. Though if the time increase is negligible I'd get behind running it in watch to match production.